service/controller/app/v1/kubeconfig/kubeconfig.go

```diff
@@ -23,10 +23,10 @@ type Config struct {
 // KubeConfig service provides primitives for connecting to the Kubernetes
 // cluster configured in the kubeconfig section of the app CR.
 type KubeConfig struct {
-	g8sClient  versioned.Interface
-	k8sClient  kubernetes.Interface
-	kubeConfig kubeconfiglib.Interface
-	logger     micrologger.Logger
+	G8sClient  versioned.Interface
+	K8sClient  kubernetes.Interface
+	KubeConfig kubeconfiglib.Interface
+	Logger     micrologger.Logger
```

Comment:
Why did this become public?

Reply:
Otherwise, I don;t know how to set this up inside unittest, 
https://github.com/giantswarm/app-operator/blob/ec8c0f1b8fb6237048f7cdeb83c1325ed6dbba7f/service/controller/app/v1/resource/chart/current_test.go#L192-L196
any idea?

Reply:
Instead of making all those public, I have changed to use it `TestKubeconfig` for the testing method, is it better?